lib/ensime.coffee

```diff
@@ -163,7 +167,7 @@ module.exports = Ensime =
     @typechecking = new TypeCheckingFeature()
 
     initClient = =>
-
+      @activatedSubscriptions()
```

Comment:
I think we need one more step here - to unsubscribe the other ones so we can switch back and forth. I'm a bit busy at the moment, but will look into it tonight!